pom.xml

```diff
@@ -30,7 +30,7 @@
 		<logback.version>1.1.11</logback.version>
 		<selenium.version>3.0.0</selenium.version>
 		<build.server.version>1.2.0</build.server.version>
-		<git.server.version>1.0.10</git.server.version>
+		<git.server.version>1.0.11-SNAPSHOT</git.server.version>

```

Comment:
Actually this should only change in your feature branch, and not in this commit. Can you undo this change here?